src/main/kotlin/lotto/Application.kt

```diff
@@ -3,10 +3,18 @@ package lotto
 fun main() {
```

Comment:
Why don't we check the comments from previous PR and see if we can apply some of them?
I see that the schedule seems pretty tight, so give it a try if it's possible, but don't stress it.